src/coreclr/vm/ceeload.h

```diff
@@ -1361,7 +1366,9 @@ class Module : public ModuleBase
     void NotifyEtwLoadFinished(HRESULT hr);
 
     // Enregisters a VASig.
-    VASigCookie *GetVASigCookie(Signature vaSignature);
+    VASigCookie *GetVASigCookie(Signature vaSignature, const SigTypeContext* typeContext);
+private:
+    static VASigCookie *GetVASigCookieWorker(Module* pDefiningModule, Module* pLoaderModule, Signature vaSignature, const SigTypeContext* typeContext);
```

Comment:
If this is static, does it really need to be defined in the header?Are we using any private members on `Module`? If not please remove from header and mark the function in the translation unit as `static`.

Reply:
Yes, we are using private members on `Module` in `GetVASigCookieWorker ` so it has to be defined in the header.